### ISSUE TITLE
.ui-selectmenu is still tab-able after calling .selectmenu('disable')

### ISSUE THREAD

**Issue by ianthedev** [fnagel/jquery-ui#301]
After calling `.selectmenu('disable')`, the `tabindex` of `a.ui-selectmenu` is still `0` so it is still tab-able.

**Comment by fnagel** [comment]
Problem confirmed.

**Comment by fnagel** [comment]
This issue should be fixed in latest commit. Please test and give feedback!

**Comment by fnagel** [comment]
Any feedback on this issue?

**Comment by fnagel** [comment]
Closed. Please re-open if needed.
